src/getData.js

```diff
@@ -22,6 +22,9 @@ const getData = () => {
         const newLi = document.createElement('li');
         ulWrap.appendChild(newLi);
         document.querySelectorAll('.score-list > li')[i].innerText = `${arrOfScores[i].user} : ${arrOfScores[i].score}`;
+        if (i % 2 === 0) {
+          document.querySelectorAll('.score-list > li')[i].style.backgroundColor = '#bdbdbdb3';
+        }
       }
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to **validate** your data against _repetitive input_, _numbers used as names_, etc

**Repetitive entries:**
![image](https://user-images.githubusercontent.com/12745474/136416322-af40670f-cb18-4f0a-b06d-636cacc573ea.png)